USAGE.md

```diff
@@ -47,7 +47,8 @@ You should get a meaningful error message if the authentication fails.
 
 Instead of using echo statements, you can enable/disable debugging
 with the `$wiki->debugMode($boolean)` method.
-Currently only output from the logon process is printed for debugging.
+Debug logging is printed to [standard output](https://en.wikipedia.org/wiki/Standard_output)
+and includes API requests/responses as well as retry messages for lag errors.
```

Comment:
```suggestion
and includes API requests/responses as well as retry messages for [lag errors](#maximum-lag-and-retries).
```

Reply:
Also, what do you think about moving this content to a dedicated section under "Other stuff", perhaps right before the "Handling errors and exceptions" section?

Reply:
Good idea, done (after it).